examples/minimal/module/rust/src/lib.rs

```diff
@@ -0,0 +1,43 @@
+//
+// Copyright 2018 The Project Oak Authors
```

Comment:
nit: should probably be 2020 (also elsewhere 2019 is used in new files)

Reply:
Heh.. Time flies when you're having fun.